app/schema/index.ts

```
@@ -0,0 +1,92 @@
+
+const yearSchema = z.number().int();
+const monthSchema = z.number().int().min(1).max(12);
+const attendancesSchema = z
+  .object({
+    date: z
+      .string()
+      .min(1)
+      .refine(
+        (val) => {
+          const date = new Date(val);
+          return !isNaN(date.getTime());
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)